### DISCUSSION TITLE
Is there a way to describe a window as "AlwaysOnTop"?

### DISCUSSION THREAD

**Discussion by gbraad** [c-smile/sciter-js-sdk#261] [Q&A]
I made a simple desktop clock that is hovering over other windows. However, when clicked on a window that would overlap, it disappear according to a lower z-order. Is there a way to describe the window as "AlwaysOnTop" ?

![image](https://user-images.githubusercontent.com/1894/148633023-3ccb9ffe-f9c6-4daf-bb8d-e7670131960b.png)

**Comment by pravic** [comment]
[`window.isTopmost`](https://github.com/c-smile/sciter-js-sdk/blob/main/docs/md/Window.md)

**Comment by gbraad** [comment]
I have tried this, but didnt have the desired effect. Or is the following not possible? 

```
<html window-isTopmost="true">

```

**Comment by c-smile** [comment]
```HTML
<script>
   Window.this.isTopmost = true;
</script>
```

**Comment by gbraad** [comment]
Is there documentation that describes what is and isn't possible from the `<html window-...>` tag?

**Comment by gbraad** [comment]
The issue is resolved, but the naming `isAlwaysTopmost` would have made more sense, plus the difference what can and can not be in the `html` window definition.

**Reply by c-smile** [reply]
> naming isAlwaysTopmost would have made more sense

Sciter cannot guarantee that "Always". 

What if,  after your call, other window in the system will also want to be topmost ?

**Comment by c-smile** [comment]
https://github.com/c-smile/sciter-js-sdk/blob/main/docs/md/HTML.md#window-attributes
